docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command:
+      sh -c "python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
The `command` for the `cinema` service runs migrations and then starts the server. Consider adding error handling or retry logic for the migration step to handle potential failures gracefully.